src/dhtproto/node/request/Get.d

```diff
@@ -102,11 +107,13 @@ public abstract scope class Get : SingleKey
         Params:
             channel_name = name of channel to query
             key = key of record to find
+            value_getter_dg = The delegate that is called with the value.
 
         Returns:
             value of queried record, empty array if not found
 
     ***************************************************************************/
 
-    abstract protected Const!(void)[] getValue ( cstring channel_name, cstring key );
+    abstract protected void getValue ( cstring channel_name, cstring key,
+        void delegate ( const(void)[] ) value_getter_dg );
```

Comment:
Should be a scope delegate to avoid closure allocations.
Same for all delegates in this PR.

Reply:
Fixed!